dashboard/cron.py

```diff
@@ -289,10 +289,10 @@ def update_with_bq_access(self):
             job_config.query_parameters = query_params
 
             # Location must match that of the dataset(s) referenced in the query.
-            bq_query = bigquery_client.query(final_bq_query, location='US', job_config=job_config)
-
-            resource_access_df: pd.DataFrame = bq_query.to_dataframe()
-            total_bytes_billed += bq_query.total_bytes_billed
+            bq_job = bigquery_client.query(final_bq_query, location='US', job_config=job_config)
```

Comment:
Probably this is step the rate limit error happens. just an observation probably since the error is raised to the outer function where this method is called from. 

Optional: can we wrap this line `try-except` and raises exception exclusivity not sure what a best practice here since it will work either way

Reply:
Yeah, I originally had it localized with a try/except but I didn't know what specific action to take that the outer one wouldn't just handle. It basically just logged a message and Raised the exception. 

It has to either get an Exception out of this method or return a tuple that indicates status. Seems more pythonic to just let it do the Exception.